Common/MVKCommonEnvironment.h

```diff
@@ -96,6 +96,10 @@ extern "C" {
 #	define MVK_XCODE_13 			((__MAC_OS_X_VERSION_MAX_ALLOWED >= 120000) || \
 									 (__IPHONE_OS_VERSION_MAX_ALLOWED >= 150000))	// Also covers tvOS
 #endif
+#ifndef MVK_XCODE_12_2
+#	define MVK_XCODE_12_2 			((__MAC_OS_X_VERSION_MAX_ALLOWED >= 110000) || \
```

Comment:
Is `MVK_XCODE_12_2` needed? The code here that uses it seems to be all macOS, so I'm not understanding the need to distinguish iOS 14.2. 

And on macOS builds, `MVK_XCODE_12_2` does exactly what `MVK_XCODE_12` does, so I don't see a need for `MVK_XCODE_12_2` as a separate definition from `MVK_XCODE_12`.

The reason `MVK_XCODE_12` uses `101600` is because earlier versions of Xcode 12 used `101600` before Apple made the decision to follow macOS `10.15` with macOS `11`. We could potentially change `MVK_XCODE_12` to use `110000` now, strictly for developer clarity, but the behavior is the same leaving it as `101600`.

Reply:
Ah, I did not know that 10.16 is actually 11.0, and thought they were actual distinct versions, which is the sole reason why I added a separate macro.  I guess I can just use MVK_XCODE_12 instead.